pkg/utils/utils.go

```diff
@@ -445,6 +445,12 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 	}
 	// set PF mtu
 	if iface.Mtu > 0 && iface.Mtu > ifaceStatus.Mtu {
+		if iface.ExternallyManaged {
```

Comment:
Thanks for adding this. We won't rely only on the webhook anymore.

Just one question, this will prevent the configuration but how can we avoid `NeedUpdate` to return `true` and force a node drain when this is the only change? We will need a check there to prevent the draining.

Reply:
good question,

it seems we always need to drain in case of externally managed since even if VF are there we still need to set admin mac / node&port guid

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/b393f307f037b3bced2392e7dfbab3c85d18b372/pkg/utils/utils.go#L393

Reply:
right, we could do it for the dpdk case though. Maybe in a future pr.